assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -38,6 +38,10 @@ function useUploadVideoFrame({ updateMediaElement }) {
   }));
   const setProperties = useCallback(
     (id, properties) => {
+      // properties is a function in this case, which will carry the poster
+      // value in the return object.
+      // function updateElementsByResourceId is defined in
+      // reducers/updateElementsByResourceId.js
```

Comment:
This comment seems unnecessary.

Reply:
Sure, I will remove the first two lines, but I would prefer to keep the file pointer for the following engineers, as it will be hard to find without enough background knowledge.

Reply:
Why was this hard to find? A simple text search in VSCode showed it.